source/math/FloatVectorList.ooc

```diff
@@ -247,15 +235,97 @@ FloatVectorList: class extends VectorList<Float> {
 		tempVector free()
 		result
 	}
+	/* calculate median without copying and sorting the vector
+			WARNING: this function can partially rearange the elements in vector
+	*/
+	fastMedian: func (start := 0, end := -1) -> Float {
+		if (end < 0)
+			end := this count - 1
```

Comment:
I should just remove it, for some reason the vector test failed when the 'end' defaulted to 'this count - 1'. I just assumed that this is just another weird ooc compiler behaviour. Now after running 'rock -x' and compiling again it worked ok. I will correct this.